openedx/features/redhouse_panel/api/permissions.py

```diff
@@ -0,0 +1,11 @@
+from rest_framework.permissions import BasePermission
+
+from openedx.features.redhouse_panel.constants import REDHOUSE_PANEL_GROUP_NAME
+
+
+class CanAccessRedhousePanel(BasePermission):
+
+    def has_permission(self, request, view):
+        if request.user.is_superuser:
+            return True
+        return request.user.groups.filter(name__in=[REDHOUSE_PANEL_GROUP_NAME]).exists()
```

Comment:
can we make a util function for this permission and `has_panel_permission()` as both are doing the same job? This way we are creating 2 copies for the same logic.

Reply:
Created a new util `has_panel_permission`. Done